### ISSUE TITLE
Timezone

### ISSUE THREAD

**Issue by pebau** [FAIRiCUBE/data-requests#60]
Question to use case providers: How to handle timezones?

Issue is practically relevant when time resolution is days or higher.

- Q1: are you concerned about the fact that timestamps may be up to 12h off what you expect?
- Q2: How to deal with data stretching timezones AND having a relevant fine-grain resolution? Example: [A number of C3S and ECMWF datasets have resolution down to hours](https://confluence.ecmwf.int/pages/viewpage.action?pageId=197702790) and stretch across European timezones. Without timezones, noon is not noon when going from Portugal to Poland.

**Comment by robknapen** [comment]
My input on this: Yes, some types of data need sufficient attention to the time zones. I think they should then be included in the time stamp and properly processed. Either automatically by the data service (i.e. request the data converted to a specified time zone), or by the user. Handling leap years can be a related challenge.

**Comment by KathiSchleidt** [comment]
Agreed, time zones are definitely relevant! 
At present, we're dealing with daily values, but I see UC where hourly values would be relevant (e.g. temperature, relevant when during the day this was)
For the case of datasets spanning multiple time zones (e.g. all of EU), the user may need to take this into account when processing the data. Can rasdaman adjust the time to a requested time-zone, or is this on the user

To `Q1: are you concerned about the fact that timestamps may be up to 12h off what you expect?`, not sure what you're pointing to here, but I'd definitely be concerned if the timestamps are off (think this 12h shift is to do with the pixel-is-point vs. pixel-is-area aspect I've brought into #59

**Comment by pebau** [comment]
indeed, as one facet in the larger investigation on time we are working on a concept that will automatically convert timezones if (!) these are properly indicated during ingestion.

I will be curious about the UC opinion on what means "off" and what means "on spot".

**Comment by pebau** [comment]
@robknapen

> Either automatically by the data service (i.e. request the data converted to a specified time zone), 

- we cannot: automatically determine the timezone during ingestion, 
- we can: transform to some desired timezone during retrieval.

**Comment by KathiSchleidt** [comment]
I will be curious about the UC opinion as this differs per dataset!

In addition, as you're ignoring value-is-pixel (e.g. value valid for the entire temporal unit) - is this not relevant here?
